### PR TITLE
[DASH-2242] Change "Choose files from disk" into white one

### PR DESCRIPTION
**Pull request by Flixow**

### CODE REVIEW

File: src/apps/Hosting/UploadFilesButton.js [hunk 1]
```diff
@@ -32,9 +33,14 @@ class UploadFilesButton extends Component {
   }
 
   render = () => {
-    const inputStyles = {
-      display: 'none',
-      position: 'absolute'
+    const styles = {
+      inputStyles: {
```
Comment (wpasinski) [inline src/apps/Hosting/UploadFilesButton.js]: `input`

File: src/apps/Hosting/UploadFilesButton.js [hunk 2]
```diff
@@ -32,9 +33,14 @@ class UploadFilesButton extends Component {
   }
 
   render = () => {
-    const inputStyles = {
-      display: 'none',
-      position: 'absolute'
+    const styles = {
```
Comment (wpasinski) [inline src/apps/Hosting/UploadFilesButton.js]: this should be a separate method since there are multiple properties.

**Comment by patrykkopycinski** [comment]
@Flixow screenshot

File: src/apps/Hosting/UploadFilesButton.js (hunk 2 above)
Comment (patrykkopycinski) [inline src/apps/Hosting/UploadFilesButton.js]: **ALWAYS** styles should be declared outside the `render` method

File: src/apps/Hosting/UploadFilesButton.js [hunk 3]
```diff
@@ -32,11 +37,8 @@ class UploadFilesButton extends Component {
   }
 
   render = () => {
```
Comment (patrykkopycinski) [inline src/apps/Hosting/UploadFilesButton.js]: don't use arrow function in `render`

**Pull request merged**